doc/xml/user-guide.xml

```diff
@@ -626,6 +627,7 @@
             <backrest-config-option section="global" key="repo1-type">azure</backrest-config-option>
             <backrest-config-option section="global" key="repo1-path">/{[azure-repo]}</backrest-config-option>
             <backrest-config-option section="global" key="repo1-azure-account">{[azure-account]}</backrest-config-option>
+            <backrest-config-option if="'{[azure-local]}' ne 'shared'" section="global" key="repo1-azure-key-type">{[azure-key-type]}</backrest-config-option>
```

Comment:
Does this `if="'{[azure-local]}' ne 'shared'"` make sense - I thought `azure-local` was a boolean variable? Shouldn't it be `if="'{[azure-key-type]}' ne 'shared'"`?

Reply:
Whoops! Fixed.